ref-arch/real-time-analytics.rst

```diff
@@ -0,0 +1,509 @@
+.. _introduction:
+
+Real Time Analytics
+#####################
+
+Over the last few years we've helped many different kinds of clients use Citus and noticed
+a technical problem many businesses have: running real-time analytics over large streams
+of data.
+
+For example, say you're building an HTTP analytics dashboard. You have enough clients that
+around every millisecond a user hits one of their websites and sends a log record to you.
+You want to ingest all of those records (1000 inserts/sec) and create a dashboard which
+shows your clients things like how many requests to their sites are errors. It's important
+that this data show up with as little latency as possible so your clients can fix problems
+with their sites. It's also useful to show graphs of historical data, however keeping all
+the raw data around forever is prohibitively expensive.
+
+Or maybe you're building an advertising network and want to show clients clickthrough
```

Comment:
"Or" is generally not a suitable word for starting a paragraph or a sentence.

Maybe:
"Another example is: You're building an advertising network and ..."